event-handlers/src/main/java/no/sikt/nva/nvi/events/db/DynamoDbEventToQueueHandler.java

```diff
@@ -0,0 +1,83 @@
+package no.sikt.nva.nvi.events.db;
+
+import static no.unit.nva.commons.json.JsonUtils.dtoObjectMapper;
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent;
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent.DynamodbStreamRecord;
+import java.util.List;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+import no.sikt.nva.nvi.common.queue.NviQueueClient;
+import no.sikt.nva.nvi.common.queue.NviSendMessageBatchResponse;
+import no.sikt.nva.nvi.common.queue.NviSendMessageResponse;
+import no.sikt.nva.nvi.common.queue.QueueClient;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.attempt.Failure;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DynamoDbEventToQueueHandler implements RequestHandler<DynamodbEvent, Void> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DynamoDbEventToQueueHandler.class);
+    private static final int BATCH_SIZE = 10;
+    private static final String DB_EVENTS_QUEUE_URL = "DB_EVENTS_QUEUE_URL";
+    private static final String FAILURE_MESSAGE = "Failure while sending database events to queue";
+    private static final String FAILED_RECORDS_MESSAGE = "Failed records: {}";
+    private static final String INFO_MESSAGE = "Sent {} messages to queue. Failures: {}";
+    private final QueueClient<NviSendMessageResponse, NviSendMessageBatchResponse> queueClient;
+    private final String queueUrl;
+
+    @JacocoGenerated
+    public DynamoDbEventToQueueHandler() {
+        this(new NviQueueClient(), new Environment());
+    }
+
+    public DynamoDbEventToQueueHandler(QueueClient<NviSendMessageResponse, NviSendMessageBatchResponse> queueClient,
+                                       Environment environment) {
+        this.queueClient = queueClient;
+        this.queueUrl = environment.readEnv(DB_EVENTS_QUEUE_URL);
+    }
+
+    @Override
+    public Void handleRequest(DynamodbEvent input, Context context) {
+        attempt(() -> {
+            splitIntoBatchesAndSend(input);
```

Comment:
See attached diagram in jira task @truhacevkir

Reply:
I saw it yes, so we keep that 10 documents til we get to createIndexDocumentHandler, but if something goes wrong here or before we get to createIndexDocumentHandler, we loose all documents from eventen, instead of loosing just single record

Reply:
Even if we put 10 into the sqs, this is 10 individual items, so if something fails futher down the line, they are not strongly bundled and we have the possibility to fail single items? or at least I think it should be possible? Also the only thing it does here is to add to sqs, so I would argue that the risk is very small anyway. It will probably be only 10 in and 10 out here, so only one batch sent, is that correct Mona?  In this first step after dynamodb here I think its a big advantage with batches as its hard to scale the dynamodblisteneres and it should be as fast as possible

Reply:
The plan is 10 in and 10 out, yes. But implemented it to handle more then 10 in as well. But i get Kirs point, if some of the batch items fail, we need to handle this as well. Suggestion: Send failed batch entries to dlq with some kind of retry-tool? Lets have a quick call.

Reply:
Dere har sikkert en plan, jeg tenker bare at det blir litt mer problematisk i finne ut av ting når hele batchen feiler og vi må grave gjennom alle entries'ene i den.